packages/angular-cli/blueprints/ng2/files/__path__/app/app.component.spec.ts

```diff
@@ -13,21 +13,21 @@ describe('App: <%= jsComponentName %>', () => {
   });
 
   it('should create the app', async(() => {
-    let fixture = TestBed.createComponent(AppComponent);
-    let app = fixture.debugElement.componentInstance;
+    let fixture:ComponentFixture<AppComponent> = TestBed.createComponent(AppComponent);
```

Comment:
There should be a space here and on all following types or the linting will fail.